landoui/forms.py

```diff
@@ -90,3 +93,83 @@ class UserSettingsForm(FlaskForm):
         ]
     )
     reset_phab_api_token = BooleanField('Delete', default="")
+
+
+class YesNoField(RadioField):
+    """
+    A simple boolean field displayed as a list of radio buttons
+    Will output a bool value (or None on unknown)
+    """
+
+    def post_validate(self, *args, **kwargs):
+        values = {'yes': True, 'no': False, 'unknown': None}
+        self.data = values.get(self.data)
+
+
+class ApprovalRequestForm(FlaskForm):
```

Comment:
Can we rename this to something uplift-specific, such as `UpliftApprovalForm`?  There is also a sec-approval request form and the generic name `ApprovalRequestForm` doesn't specify which workflow `lando.forms.ApprovalRequestForm` is used in.